worker.sh

```diff
@@ -195,57 +244,39 @@ start_k8s() {
             --network-plugin-dir=/etc/cni/net.d
 }
 
-configure_docker(){
-	# Configure docker settings, then restart it
-    case "${lsb_dist}" in
-        amzn)
-            DOCKER_CONF="/etc/sysconfig/docker"
-            sed -i.bak 's/^\(MountFlags=\).*/\1shared/' $DOCKER_CONF
-            service docker restart
-            ;;
-        centos)
-            DOCKER_CONF="/usr/lib/systemd/system/docker.service"
-            sed -i.bak 's/^\(MountFlags=\).*/\1shared/' ${DOCKER_CONF}
-            # Add support for docker registry
-            if [[ -n ${DOCKER_REGISTRY_URL} ]]; then
-			  sed -i "/^ExecStart=/ s~$~ --insecure-registry=${DOCKER_REGISTRY_URL}~" ${DOCKER_CONF}
-			fi
-            systemctl daemon-reload
-            systemctl restart docker
-            ;;
-        ubuntu|debian)
-            if command_exists systemctl; then
-              DOCKER_CONF=$(systemctl cat docker | head -1 | awk '{print $2}')
-              sed -i.bak 's/^\(MountFlags=\).*/\1shared/' $DOCKER_CONF
+start_service_k8s(){
+		echo "Installing k8s as a service ..."
+		docker run -d \
+		    --volume=/:/hostfs:rw \
+		    --net=host \
+		    --pid=host \
+		    --privileged \
+		    taimir93/hyperkube-amd64:v1.3.0-beta.2 \
```

Comment:
you can use fest (fujitsu est) orgainization. register on docker hub, add your company email to your profile and I grant you access